design/object-bucket/object-bucket.md

```diff
@@ -61,7 +58,14 @@ _As an admin, I want to expose an existing Ceph object store to users so they ca
 
 ![admin-actions](./cobc-admin.png)
 
-1. The admin creates a StorageClass with the `provisioner=ceph.rook.io/object`, `parameters[serviceName]=OBJECT-STORE-SERVICE`, and `parameters[serviceNamespace]=OBJECT-STORE-NAMESPACE` key-values.
+1. The admin creates a [StorageClass](#storageclass) with the `provisioner=ceph.rook.io/object`, `parameters[serviceName]=OBJECT-STORE-SERVICE`, and `parameters[serviceNamespace]=OBJECT-STORE-NAMESPACE` key-values.
```

Comment:
What do you think about moving this use case to an Appendix or somewhere out of the main PR flow?

Reply:
I'm not sure about this as it's step 0 of the system's workflow.